core/evm.go

```diff
@@ -107,6 +108,46 @@ func GetHashFn(ref *block.Header, chain ChainContext) func(n uint64) common.Hash
 	}
 }
 
+// GetVRFFn returns a GetVRFFn which retrieves header vrf by number
+func GetVRFFn(ref *block.Header, chain ChainContext) func(n uint64) common.Hash {
```

Comment:
do you have any limit on n?  what if n is a huge number?

Reply:
The limit is 256 blocks, it's using similar logic as GetHashFn